other/react-native/__tests__/render-tests.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable react/prop-types */
+// eslint-disable-next-line import/no-unassigned-import
+import {Text, TextInput, View} from 'react-native'
+import React from 'react'
+
+// Note: test renderer must be required after react-native.
+import TestRenderer from 'react-test-renderer'
+
+import Downshift from '../../../dist/downshift.native.cjs'
+
+test('renders with React Native components', () => {
+  const RootView = ({innerRef, ...rest}) => <View ref={innerRef} {...rest} />
+  const renderSpy = jest.fn(({getRootProps, getInputProps, getItemProps}) => (
+    <RootView {...getRootProps({refKey: 'innerRef'})}>
+      <TextInput {...getInputProps()} />
+      <View>
+        <Text {...getItemProps({item: 'foo', index: 0})}>foo</Text>
+        <Text {...getItemProps({item: 'bar', index: 1})}>bar</Text>
+      </View>
+    </RootView>
+  ))
+  const element = <Downshift render={renderSpy} />
+  const renderer = TestRenderer.create(element)
+  expect(renderSpy).toHaveBeenCalledWith(
+    expect.objectContaining({
+      isOpen: false,
+      highlightedIndex: null,
+      selectedItem: null,
+      inputValue: '',
+    }),
+  )
+  const tree = renderer.toJSON()
+  expect(tree).toMatchSnapshot()
```

Comment:
We _could_ remove this snapshot, but I think ensuring that we render the basic set of React Native components here, while applying the functions from Downshift, is a fair trade-off for using a snapshot.